bootstrap/integration/checkout_integration_test.go

```diff
@@ -103,7 +103,7 @@ func TestCheckingOutLocalGitProject(t *testing.T) {
 
 	env := []string{
 		"BUILDKITE_GIT_CLONE_FLAGS=-v",
-		"BUILDKITE_GIT_CLONE_MIRROR_FLAGS=--bare",
+		"BUILDKITE_GIT_CLONE_MIRROR_FLAGS=--config pack.threads=35",
```

Comment:
Let's get your use-case in the test suite 😁

Reply:
Ha ha ha, awesome!